sdk/python/foundation-models/benchmarking/evaluating_claude_models.ipynb

```diff
@@ -0,0 +1,205 @@
+    "from azure.identity import DefaultAzureCredential\n",
+    "from azure.ai.ml import MLClient\n",
+    "\n",
+    "# client for AzureML Workspace actions\n",
+    "ml_client = MLClient(\n",
+    "    credential=DefaultAzureCredential(),\n",
+    "    subscription_id=subscription_id,\n",
+    "    resource_group_name=resource_group,\n",
+    "    workspace_name=workspace_name,\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2e4bfaa2-fd99-4850-a9c4-8838c4ef9313",
+   "metadata": {},
+   "source": [
+    "Finally, to launch the evaluation pipeline job, run the following:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3e4c5468-7585-481e-bb20-ad91373cac6c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from azure.ai.ml import load_job\n",
+    "\n",
+    "# load the pipeline from the yaml def\n",
+    "pipeline_job = load_job(f\"./evaluation_pipelines/claude-2_1/{eval_name}.yaml\")\n",
+    "\n",
+    "# Set pipeline job inputs\n",
+    "pipeline_job.settings.default_compute = compute_name\n",
+    "pipeline_job.inputs.endpoint_url = bedrock_endpoint_url\n",
+    "pipeline_job.inputs.ws_connection_name = connection_name\n",
+    "pipeline_job.inputs.sample_ratio = sample_ratio\n",
+    "\n",
+    "# Start the job in the Workspace\n",
+    "returned_job = ml_client.jobs.create_or_update(\n",
+    "    pipeline_job, experiment_name=experiment_name\n",
+    ")\n",
+    "returned_job"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "095005d9-1deb-4097-a718-4059a335ec09",
+   "metadata": {},
+   "source": [
+    "Run the next cell to stream the job. Notebook execution will be paused until the job finishes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1376aa46-725d-4d6c-8725-3a2d4d52bdfc",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Wait until the job completes\n",
+    "ml_client.jobs.stream(returned_job.name)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "07fa2c79-f875-4c4f-8f09-37158b56bdcd",
+   "metadata": {},
+   "source": [
+    "## Retrieve accuracy scores from the run\n",
+    "When the pipeline finishes, you can retrieve evaluation metrics from the run via mlflow. The primary measure of accuracy for the evals is `mean_exact_match`, with the exception of human_eval which uses `pass@1`. \n",
+    "\n",
+    "Mean exact match is the proportion of model predictions that exactly match the corresponding correct answers. Thus, it is applicable to question answering evaluations that are multiple choice or have a single, correct answer. The pass@1 metric is used for evaluating code generation and is the proportion of model generated code solutions that pass a set of unit tests given in the eval dataset. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "299bf8ea-49ab-44c3-96e6-51902fa03df4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import mlflow\n",
```

Comment:
Here should we add some comment/cell suggesting trying other datasets as well?

Reply:
Not sure what you mean exactly - but the list of evals should make it clear that users can try several different datasets.